bundles/CoreBundle/src/Migrations/Version20210928135248.php

```diff
@@ -29,7 +29,9 @@ public function getDescription(): string
 
     public function up(Schema $schema): void
     {
-        $schema->dropTable('sanitycheck');
+        if($schema->hasTable('sanitycheck')) {
```

Comment:
just a code style fix
```suggestion
        if ($schema->hasTable('sanitycheck')) {
```
and not related to the change of this PR, but casually noticed that in the down
https://github.com/pimcore/pimcore/blob/83e0785ab177c9166a20b28511fbf2910f328340/bundles/CoreBundle/src/Migrations/Version20210928135248.php#L39-L40
there is redundancy, either `IF NOT EXISTS` or !`$schema->hasTable` should go

Reply:
You are right, I fixed the `down()` method too.